README.md

```diff
@@ -153,6 +153,8 @@ And it should show no changes
 ### Deployment
 
 You can deploy this project using the following link:
+
+https://evillaz.github.io/my-portfolio/
```

Comment:
- [ ] As suggested in the project requirements, I think it would be better to deploy your portfolio using GitHub pages, and kindly remember to provide a valid link to the live version of your project in the README. :pray:

    _you may try to follow this [tutorial](https://www.youtube.com/watch?v=M5mg0r4ajt4) for guidance 😎😎😎_